tcms/static/js/tcms_actions.js

```diff
@@ -119,8 +119,7 @@ var default_messages = {
     'last_case_run': 'It is the last case run',
     'bookmark_added': 'Bookmark added.',
     'no_run_selected': 'No run selected.',
-    'invalid_bug_id': 'Please input a valid bug id which is less equal 7 numberes, e.g. 1001234',
-    'invalid_jira_id': 'Please input a valid JIRA id, e.g. TCMS-32',
+    'invalid_bug_id': 'Please input a valid bug id!',
```

Comment:
When user enters incorrect bug id of Bugzilla or issue key of JIRA, Nitrate should show original message.